.github/workflows/regression-report.yml

```diff
@@ -0,0 +1,248 @@
+#####################################################################################
+# GitHub Action to generate Checkstyle report.
+#
+# Workflow starts when:
+# 1) issue comment - created
+#
+# Requirements:
+# 1) secrets.AWS_ACCESS_KEY_ID - access key for AWS S3 service user
+# 2) secrets.AWS_SECRET_ACCESS_KEY - security access key for AWS S3 service user
+#
+# If you need to change bucket name or region, change AWS_REGION and AWS_BUCKET_NAME variables.
+# For another bucket, you will need to change the secrets.
+#####################################################################################
+name: regression-report
+env:
+  AWS_REGION: us-east-2
+  AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  USER_LOGIN: ${{ github.event.issue.user.login }}
+
+on:
+  issue_comment:
+    types: [ created, edited ]
```

Comment:
Lets remove "edited", I am afraid that it will be too much types of events to handle (we already have long queue for actions) and 
we use "github.run_id" for concurrent, so it will be not clear why some reports generated, as we will not see easily first version of comment.

We can reconsider it later on.

Reply:
https://github.com/checkstyle/test-configs/issues/121

Reply:
https://github.com/checkstyle/test-configs/issues/122

Reply:
> Lets remove "edited", I am afraid that it will be too much types of events to handle (we already have long queue for actions) and we use "github.run_id" for concurrent, so it will be not clear why some reports generated, as we will not see easily first version of comment.
> 
> We can reconsider it later on.

Done